README.md

```diff
@@ -10,12 +10,19 @@ This repository hosts the source code and releases of the new CLI.
 
 To install the CLI, use either [Homebrew](https://brew.sh/) (on Linux, macOS, or the Windows Subsystem for Linux) or [Scoop](https://scoop.sh/) (on Windows):
 
-### HomeBrew
+### Homebrew
 
 ```console
 brew install platformsh/tap/platformsh-cli
 ```
 
+After installing or updating platformsh-cli via Homebrew, make sure you have the right `platform` in your path by issuing
+```bash
+hash -r && ls -l $(which platform)
+```
+
+You should see a path that involves homebrew, `/opt/homebrew/bin/platform` on an Apple Silicon Mac (M1/M2), `/usr/local/bin/platform` on an Intel Mac, `/home/linuxbrew/.linuxbrew/bin/platform` on Linux. If you don't see that, then delete the target that you are seeing with `sudo rm $(which platform) && hash -r` and then you should see the correct `platform` tool in `which platform`.
```

Comment:
```suggestion
You should see a path that involves homebrew, `/opt/homebrew/bin/platform` on an Apple Silicon Mac (M1/M2), `/usr/local/bin/platform` on an Intel Mac, `/home/linuxbrew/.linuxbrew/bin/platform` on Linux. If you don't see that, then delete the target that you are seeing with `sudo rm "$(which platform)" && hash -r` and then you should see the correct `platform` tool in `which platform`.
```

otherwise you might be removing 3 files in the local directory: `platform`, `not`, and `found` (the first of which might be problematic, depending where you were running this)

Reply:
`which platform` does not return any output on a linux or macOS system if `platform` is not found... Where did you see "platform not found"? But they should have already succeeded in getting `which platform` before getting to this point.  But I don't object to the proposed change.